.codacy.yml

```diff
@@ -1,2 +1,20 @@
 ---
+engines:
+ pylint:
+   enabled: true
+   python_version: 3
+    
+ duplication:
+   enabled: true
+
+ metrics:
+   enabled: true
+
+ coverage:
+   enabled: true
+
+languages:
+   python:
+
 exclude_paths:
+   
```

Comment:
Add a new line at the end of the file.

Reply:
@Fenn-CS Is @MehaKaushik s review incorporated?

Reply:
Ok, will try to follow up with all of the open issues soon. Currently have exams